src/deepsparse/open_pif_paf/pipelines.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+OpenPifPafPipeline
+"""
+from typing import Type
+
+from deepsparse.open_pif_paf.schemas import OpenPifPafInput, OpenPifPafOutput
+from deepsparse.pipeline import Pipeline
+
+
+__all__ = [
+    "OpenPifPafPipeline",
+]
+
+
+@Pipeline.register(
+    task="open_pif_paf",
+    default_model_path=("/home/damian/pifpaf/openpifpaf-resnet50.onnx"),
```

Comment:
do we have a public onnx we can point to?

Reply:
yup okay, getting that for a pretrained model with an annotation example is what i'd like to have eventually